deploy/20_cloud-ingress-operator.ClusterRole.yaml

```diff
@@ -24,6 +24,7 @@ rules:
     - get
     - list
     - watch
+    - patch
     - delete
     - create
```

Comment:
Should we simplify this to `- "*"`?

Reply:
Happy to do it. Is there any specific benefit?

Reply:
no, it's just easier for humans to read :)